src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -104,14 +104,14 @@
           Condition="'$(MSBuildProjectExtension)' == '.depproj'" />
 
   <Import Project="IL.targets"
-           Condition="'$(MSBuildProjectExtension)' == '.ilproj' AND '$(SkipImportILTargets)'!='true'" />
```

Comment:
I assume this means our ilproj's stop working? If they don't work as sdk projects then we don't need this condition but if they do then we do still need the import.

Reply:
Yea, `ilproj` is a bit tricky, unfortunately.

We DO use the SDK style project for it.  However, the SDK doesn't have support for the `IL` language.  It only ships with support for C#, VB and F#. In order to make other languages work correctly, there is a property called `$(LanguageTargets)` which gets imported at the correct place.
See https://github.com/dotnet/sdk/blob/f8542e15d2dc54bc7fcce32bdf5046cde4483d9a/src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets#L28-L41.

So in the one `ilproj` we have in dotnet/corefx, I'm setting `<LanguageTargets>$(ToolsDir)\IL.targets<LanguageTargets>`, which allows it to be imported correctly with the SDK.  See https://github.com/dotnet/corefx/pull/29831/commits/be5ceb5379738fab0a071f98e63f18496bc0388a#diff-89d3c36d89fc54d71aa52b9438ac028cR8 for the corefx change.